integration-tests/config/config.json

```diff
@@ -1,203 +1,127 @@
 {
-  "repositories": [
```

Comment:
This file should be ignored as well since it is generated by the tool.

Reply:
Explicitly excluded this file